### ISSUE TITLE
Wrong window size or text size not matching in some dialogs

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#5371]
While testing the new PQ functions I realized some design glitches on my device, see screenshots attached.
AFAIR there is at least one other dialog with the same problem, but I could not find it.

```
--- System information ---
Device: GT-I9505 (jfltexx, samsung)
Android version: 5.0.1
Android build: LRX22C.I9505XXUHOJ2
Cgeo version: 2016.01.11-NB1-f1ec8b2
Google Play services: enabled
Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: rotation vector
Hide own/found: true
Map strategy: auto
HW acceleration: enabled (default state)
System language: de_DE
Geocaching sites enabled:
- geocaching.com: logged in (Login OK) / Premium
- opencaching.de: logged in (Login OK)
Installed cgeo plugins: calendar, contacts
--- End of system information ---
```

![screenshot_2016-01-10-10-56-20](https://cloud.githubusercontent.com/assets/949669/12244020/d907d698-b8a1-11e5-93ef-653573fe649e.png)
![screenshot_2016-01-10-10-58-04](https://cloud.githubusercontent.com/assets/949669/12244022/dbb8eff8-b8a1-11e5-890b-90c475b27c2b.png)

**Comment by pstorch** [comment]
I found the restore dialogues when switching between nightly and my development version:

![screenshot_2016-01-11-20-33-24](https://cloud.githubusercontent.com/assets/5598124/12244219/c6df5f3a-b8a2-11e5-8e78-2e8fbcf072dc.png)
![screenshot_2016-01-11-20-33-14](https://cloud.githubusercontent.com/assets/5598124/12244227/d16d982c-b8a2-11e5-87e8-cb4f49617f84.png)

**Comment by Lineflyer** [comment]
FYI: The gprahic glitches in the pocket query menu do no longer show up for me in 2016.01.16-NB

**Comment by pstorch** [comment]
The PQ Dialogs are fixed some time ago, the Restore Dialogs are still out of shape.
And today I found a new one:
When long clicking on the "Stored" icon on the main screen and the select `<Create new list>`
![Uploading 2016-03-23 20.20.26.png…]()

**Comment by kumy** [comment]
@Lineflyer assigned for testing.

**Comment by Lineflyer** [comment]
Confirmed fixed (on my device Samsung s4) with 2016.04.03-NB
